sqlglot/expressions.py

```diff
@@ -1589,6 +1597,31 @@ class Min(AggFunc):
     pass
 
 
+class MixedTypeAdd(Func):
+    arg_types = {
+        "this": True,
+        "expression": True,
+        "unit": False,
+        "output_format": False,
```

Comment:
call this format for consistency?

Reply:
I actually explicit added the `output_` prefix since I found it confusing when it was just `format`. The reason is that with just format it is not clear if you are providing the format for the provided date or the format of the output. After considering that if you still want `format` then I am happy to change.

Reply:
ok that's fine